resources/wikia/modules/krux.js

```diff
@@ -0,0 +1,64 @@
+/*global Krux,define*/
+window.Krux || ((Krux = function () {
+	Krux.q.push(arguments);
+}).q = []);
+
+define('wikia.krux', ['wikia.window'], function (win) {
+	'use strict';
+
+	var maxNumberOfKruxSegments = 27;
+
+	function load(confid) {
```

Comment:
I don't know Krux internals and if we have to load script on every article view, but...
We don't want to append another `<script>` tag on every article view, now do we? They would accumulate.